docs/content/ingestion/hadoop.md

```diff
@@ -198,6 +198,7 @@ The tuningConfig is optional and default parameters will be used if no tuningCon
 |useExplicitVersion|Boolean|Forces HadoopIndexTask to use version.|no (default = false)|
 |logParseExceptions|Boolean|If true, log an error message when a parsing exception occurs, containing information about the row where the error occurred.|false|no|
 |maxParseExceptions|Integer|The maximum number of parse exceptions that can occur before the task halts ingestion and fails. Overrides `ignoreInvalidRows` if `maxParseExceptions` is defined.|unlimited|no|
+|useYarnRMJobStatusFallback|Boolean|If the Hadoop jobs created by the indexing task are unable to retrieve their completion status from the JobHistory server, and this parameter is true, the indexing task will try to fetch the application status from `http://<yarn-rm-address>/ws/v1/cluster/apps/<application-id>`, where `<yarn-rm-address>` is the value of `yarn.resourcemanager.webapp.address` in your Hadoop configuration. This flag is intended as a fallback for cases where an indexing task's jobs succeed, but the JobHistory server is unavailable, causing the indexing task to fail because it cannot determine the job statuses.|no (default = true)|
```

Comment:
If this defaults to true, any reason to have it be an option at all? Would anyone _not_ want this to happen on fallback?

Reply:
I was thinking maybe someone wasn't using YARN, so they'd want to disable this

Reply:
only reason I didn't suggest is because I am unsure what all circumstances could cause  `job.waitForCompletion(..)` throw IOException instead of just unavailability of JobHistory server. for someone it might increate the time to failure due to extra tries on RM and someone might not want that.
that said, main reason is probably that the work is already done :)